src/java/org/apache/cassandra/db/marshal/VectorType.java

```diff
@@ -359,6 +365,13 @@ private <V> void checkConsumedFully(V buffer, ValueAccessor<V> accessor, int off
             throw new MarshalException("Unexpected " + remaining + " extraneous bytes after " + asCQL3Type() + " value");
     }
 
+    @Override
+    public ByteBuffer getMaskedValue()
+    {
+        List<ByteBuffer> values = Collections.nCopies(dimension, elementType.getMaskedValue());
```

Comment:
Do we need to add a doc to tell the users how vectortype's mask_default was implemented ? let users know whether the usage expectations are met.

Reply:
I have added some doc lines and examples about how default masking values are generated.